manager/ui/war/plugins/api-manager/html/headers/community.include

```diff
@@ -71,7 +71,7 @@
                             </a>
                         </li>
                         <li class="list-group-item" id="apiman-sidebar-apis-consume" data-target="#apis-all_apis-tertiary">
-                            <a href="{{ pluginName }}/browse/apis">
+                            <a href="{{ pluginName }}/browse/apis?q=*&cp=1&ps=12">
```

Comment:
Can you explain the `cp` and `ps`?

Reply:
Current page and page size I guess

Reply:
Ah yes, so the browse/consume page uses the search endpoint.
So to avoid displaying an empty page on first page load we fetch the first page of a wildcard search (SearchBean).

```bash
q = querry
cp = currentPage
ps = pageSize 
```
This is a nice default which does not display to much data and plays nice with our 4x3 card layout.

Reply:
![image](https://user-images.githubusercontent.com/13332383/166898762-5fe0e0b7-be9e-4ca7-9adf-5b3dd5753a8d.png)

Reply:
Did you guys add that?/ I don't remember that being the behaviour Apiman had historically

Reply:
I think that is the case since a while, apiman 2.x behaves the same.
You can change the page size in the UI if there are more entries to 25, 50 and 100 I think

Reply:
Okay, let's just leave it for now. Perhaps needs to be a bit more flexible and signpost that you can search more easily.